static/js/reactions.js

```diff
@@ -198,6 +201,11 @@ exports.get_message_reactions = function (message) {
             reaction.is_realm_emoji = true;
             reaction.url = emoji.realm_emojis[reaction.emoji_name].emoji_url;
         }
+        if (reaction.title.indexOf("You (click to remove)") !== -1) {
+            reaction.class = "message_reaction reacted";
```

Comment:
Surely we can do a test that's based on whether youre user ID is in the reaction.user.id list?  This is hacky, and in particular will break if that string is internationalized.